plugins/module_utils/entity.py

```diff
@@ -411,8 +411,16 @@ def _fetch_url(
             return {"status_code": status_code}
 
         if resp_json is None:
+            try:
+                if info.get("msg")):
```

Comment:
```suggestion
                if info.get("msg"):
```

Reply:
fixed. nice catch.

Reply:
Do we really need to catch ValueError here ? Since code has significantly changed from your first draft till now.

Reply:
no not really, it reverts back to the start of this